tests/queries/0_stateless/02293_grouping_function_group_by.reference

```diff
@@ -0,0 +1,176 @@
+0	1
+1	1
+2	1
+3	1
+4	1
+5	1
+6	1
+7	1
+8	1
+9	1
+0	1	1
+1	1	1
+2	1	1
+3	1	1
+4	1	1
+5	1	1
+6	1	1
+7	1	1
+8	1	1
+9	1	1
+0	0
+0	2
+0	3
+1	2
+1	3
+2	2
+2	3
+3	2
+3	3
+4	2
+4	3
+5	2
+5	3
+6	2
+6	3
+7	2
+7	3
+8	2
+8	3
+9	2
+9	3
+0	0
+0	2
+0	3
+1	2
+1	3
+2	2
+2	3
+3	2
+3	3
+4	2
+4	3
+5	2
+5	3
+6	2
+6	3
+7	2
+7	3
+8	2
+8	3
+9	2
+9	3
+0	0
+0	1
+0	1
+0	2
+0	3
+1	2
+1	3
+2	2
+2	3
+3	2
+3	3
+4	2
+4	3
+5	2
+5	3
+6	2
+6	3
+7	2
+7	3
+8	2
+8	3
+9	2
+9	3
+0	0
+0	1
+0	1
+0	2
+0	3
+1	2
+1	3
+2	2
+2	3
+3	2
+3	3
+4	2
+4	3
+5	2
+5	3
+6	2
+6	3
+7	2
+7	3
+8	2
+8	3
+9	2
+9	3
+0	5
+0	6
+1	5
+1	6
+2	5
+2	6
+3	5
+3	6
+4	5
+4	6
+5	5
+5	6
+6	5
+6	6
+7	5
+7	6
+8	5
+8	6
+9	5
+9	6
```

Comment:
Is it correct? I would not except that `grouping(number, number % 2)` would return 5 or 6
```
-- { echoOn }
```
may make tests more readable

Reply:
In this test we have `grouping(number, number % 2) + 3`